sdk/java-sdk/src/main/java/com/akkaserverless/javasdk/action/Action.java

```diff
@@ -106,6 +107,16 @@ public void _internalSetActionContext(Optional<ActionContext> context) {
        */
       <S> Effect<S> error(String description);
 
+      /**
+       * Create an error reply.
+       *
+       * @param description The description of the error.
+       * @param status A custom gRPC status code.
+       * @return An error reply.
+       * @param <T> The type of the message that must be returned by this call.
+       */
+      <T> Effect<T> error(String description, GrpcStatus status);
```

Comment:
I guess in the future we might introduce yet another variant to also be able to control the HTTP status code (when transcoding), but this is :+1:.